Kernel/Bus/PCI/Device.h

```diff
@@ -6,6 +6,8 @@
 
```

Comment:
Please write the commit message in the imperative mood, according to the [CONTRIBUTING.md](https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy)

Also `PCI:Device` -> `PCI::Device`